website/client/src/components/groups/discovery.vue

```diff
@@ -52,9 +52,10 @@ b-dropdown(:text="$t('sort')", right=true)
           />
           <mugen-scroll
             v-show="loading"
-            :handler="fetchGuilds"
+            :handler="triggerFetchGuilds"
             :should-handle="!loading && !hasLoadedAllGuilds"
             :handle-on-mount="true"
+            :threshold="20"
```

Comment:
I think threshold should be between 0 and 1 https://github.com/egoist/vue-mugen-scroll#threshold

Reply:
oops, I mistook it with scrolling threshold ... removed it again.

Good catch :tada: